homeworks/anna.vrana_Anna1o1Vrana/13-ng-forms/ng-forms/src/app/app.component.ts

```diff
@@ -0,0 +1,58 @@
+import {Component, OnInit} from '@angular/core';
+import {FormControl, Validators} from "@angular/forms";
+import {SaveAuthService} from "./service";
+import {FormBuilder} from '@angular/forms';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent implements OnInit {
+  title = 'ng-forms';
+  authorizationForm = this.fb.group({
+    email: new FormControl('',
```

Comment:
Суть форм білдера в тому щоб прибрати трохи копіпасту з купи конструкторів
```suggestion
    email: ['', [Validators.required, Validators.email]],
```

Reply:
modified)